### DISCUSSION TITLE
Suggested: ELK stack (Elasticsearch, Logstash, Kibana)

### DISCUSSION THREAD

**Discussion by smtdev** [tteck/Proxmox#565] [Ideas]
First of all, a big THANK YOU for your work and effort on bring us these amazing 'plug-n-play' scripts! 

"ELK" is the acronym for three open source projects: Elasticsearch, Logstash, and Kibana. Elasticsearch is a search and analytics engine. Logstash is a server‑side data processing pipeline that ingests data from multiple sources simultaneously, transforms it, and then sends it to a "stash" like Elasticsearch. 

Main website: https://www.elastic.co/es/
Github repository: https://github.com/elastic
Some Docker options:

- https://github.com/deviantony/docker-elk
- https://github.com/spujadas/elk-docker

I think it would be a great idea to have this stack in LXC. 

Thank you again!

**Comment by The1Percent** [comment]
This would be an awesome addition to the helper scripts
